embedded-can/src/lib.rs

```diff
@@ -2,7 +2,15 @@
 
 #![warn(missing_docs)]
 #![no_std]
-
+// disable warning for already-stabilized features.
+// Needed to pass CI, because we deny warnings.
+// We don't immediately remove them to not immediately break older nightlies.
+// When all features are stable, we'll remove them.
+#![cfg_attr(nightly, allow(stable_features, unknown_lints))]
+#![cfg_attr(nightly, feature(async_fn_in_trait, impl_trait_projections))]
```

Comment:
these cfg_attrs do nothing without [this](https://github.com/rust-embedded/embedded-hal/blob/master/embedded-hal-async/build.rs#L16) in build.rs. 

These were for compat with older nightlies before AFIT was stable. Now that AFIT has been stable for a few rust versions since 1.75, i'd just not put them, and support stable 1.75+ only.